addons/point_of_sale/static/src/app/generic_components/orderline/orderline.xml

```diff
@@ -4,38 +4,38 @@
+                        <li class="price-per-unit">
+                            <em t-esc="line.qty" class="qty fst-normal fw-bolder me-1" /> <t t-if="line.unit" t-esc="line.unit" />
+                            <t t-if="line.price !== 0">
+                                x <s t-esc="line.oldUnitPrice" t-if="line.oldUnitPrice" />
+                                <t t-esc="line.unitPrice" /> <span t-if="line.unit"> / <t t-esc="line.unit" /></span>
+                            </t>
+                        </li>
+                        <li t-if="line.price !== 0 and line.discount and line.discount !== '0'">
+                            With a <em><t t-esc="line.discount" />% </em> discount
+                        </li>
+                        <li t-if="line.customerNote" class="customer-note w-100 p-2 my-1 rounded text-break text-bg-warning text-warning bg-opacity-25">
+                            <i class="fa fa-sticky-note me-1" role="img" aria-label="Customer Note" title="Customer Note"/>
+                            <t t-esc="line.customerNote" />
+                        </li>
+                        <t t-foreach="line.internalNote?.split?.('\n') or []" t-as="note" t-key="note_index">
+                            <li t-if="note.trim() !== ''" t-esc="note" class="internal-note ms-1 p-2 badge rounded-pill bg-info text-info bg-opacity-25" style="font-size: 0.85rem;" />
                         </t>
-                    </li>
-                    <li t-if="line.price !== 0 and line.discount and line.discount !== '0'">
-                        With a <em><t t-esc="line.discount" />% </em> discount
-                    </li>
-                    <li t-if="line.customerNote" class="customer-note w-100 p-2 my-1 rounded text-break text-bg-warning text-warning bg-opacity-25">
-                        <i class="fa fa-sticky-note me-1" role="img" aria-label="Customer Note" title="Customer Note"/>
-                        <t t-esc="line.customerNote" />
-                    </li>
-                    <t t-foreach="line.internalNote?.split?.('\n') or []" t-as="note" t-key="note_index">
-                        <li t-if="note.trim() !== ''" t-esc="note" class="internal-note ms-1 p-2 badge rounded-pill bg-info text-info bg-opacity-25" style="font-size: 0.85rem;" />
-                    </t>
-                    <li t-foreach="line.packLotLines or []" t-as="lot" t-key="lot_index" t-esc="lot" />
-                    <t t-slot="default" />
-                </ul>
+                        <li t-foreach="line.packLotLines or []" t-as="lot" t-key="lot_index" t-esc="lot" />
+                        <t t-slot="default" />
+                    </ul>
+                </div>
+                <div class="product-price d-inline-block text-end price fw-bolder flex-grow-1">
+                    <t t-if="line.price === 'free'">Free</t>
+                    <t t-else="" t-esc="line.price"/>
+                </div>
             </div>
```

Comment:
what does  this change do ?

Reply:
its just css stuff, the code was already there. I need to have flex column for orderlines